fe/fe-core/src/main/java/com/starrocks/analysis/RestoreStmt.java

```diff
@@ -175,4 +176,9 @@ public String toSql() {
         sb.append("\n)");
         return sb.toString();
     }
+
+    @Override
+    public <R, C> R accept(AstVisitor<R, C> visitor, C context) {
+        return visitor.visitRestoreStmt(this, context);
+    }
```

Comment:
should override isSupportNewPlanner() and return true/

Reply:
> should override isSupportNewPlanner() and return true/
Thank you for your feedback. I have made repairs and added override isSupportNewPlanner().